Formula/fasta.rb

```diff
@@ -2,9 +2,9 @@ class Fasta < Formula
   # cite Pearson_1990: "https://doi.org/10.1016/0076-6879(90)83007-V"
   desc "Classic FASTA sequence alignment suite"
   homepage "https://faculty.virginia.edu/wrpearson/fasta/"
-  url "https://github.com/wrpearson/fasta36/archive/v36.3.8h_11-Feb-2020.tar.gz"
+  url "https://github.com/wrpearson/fasta36/archive/v36.3.8h_04-May-2020.tar.gz"
   version "36.3.8h"
```

Comment:
```suggestion
  version "36.3.8h.2020.05.04"
```
See https://repology.org/project/fasta/versions for how other package managers have formatted this peculiar version number.